src/Core/Geometry/TriangleMesh.hpp

```diff
@@ -249,6 +250,13 @@ class IndexedGeometry : public AttribArrayGeometry
     /// \warning There is no error check on the handles attribute type.
     inline bool append( const IndexedGeometry<IndexType>& other );
 
+    const IndexContainerType& getIndices() const { return m_indices; }
```

Comment:
Other changes suggested:
 - mark `m_indices` as private
 - add access method `withLock` and `unlock`

Reply:
Yes, we should do it for this PR (but it will change a lot of file, let's postpone when the PR converges).